src/dsp_tools/commands/excel2json/new_lists/compliance_checks.py

```diff
@@ -11,9 +11,7 @@
 
```

Comment:
It looks horrible but I did not change any logic.
The changes are:
- parse and process files was taken apart.
- The construction of the `Columns` object is now earlier.
- It contains information that is used throughout. So that information is now also accessed through the object.